src/Extension/UuidExtension.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Faker\Core\Extension;
+
+/**
+ * @experimental This interface is experimental and does not fall under our BC promise
+ */
+interface UuidExtension extends Extension
```

Comment:
As discussed, lets not include this. The UUID generation is a bit flakey anyhow and only supports v3 where there are multiple versions. Lets suggest people to use an actual UUID lib to generate those